_saas-integrations/pepperjam/v1/pepperjam-v1.md

```diff
@@ -0,0 +1,170 @@
+row-usage-hog-reasons:
+  data-structure: true
+  data-volume: true
+  lots-of-full-table: false
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Querying Details      #
+# -------------------------- #
+
+## Pepperjam's attribution windows. Stitch queries for
+## this number of days during each replication job to
+## account for any updates to existing records made during
+## this time.
+
+replication-notes: true
+attribution-window: "30 days"
+attribution-is-configurable: false
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
+#      Incompatibilities     #
+# -------------------------- #
+
+## uncomment section below if integration is compatible with any Stitch destinations
+## if incompatible with multiple destinations, create a section for each destination
+
+## incompatible:
+  ## [redshift]: "always,sometimes,never"
+  ## reason: "copy" 
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+setup-steps:
+  - title: "Obtain your API key"
+    anchor: "obtain-api-key"
+    content: |
+      1. Login to you {{ integration.display_name }} account.
+      2. Hover over the **Developer Kit** menu and then click **API Keys**.
+      3. Click **Generate New Key**.
+      4. Keep note of your API Key - you will need it to complete the next step.
+  - title: "add integration"
+    content: |
+      4. In the **API Key** field, paste the API key you obtained in [step 1](#obtain-api-key).
+  - title: "historical sync"
+  - title: "replication frequency"
+  - title: "track data" ## remove this if the integration doesn't support at least table selection
+
+
+# -------------------------- #
+#      Replication Info      #
+# -------------------------- #
+
+replication-sections:
+  - title: "Attribution windows and data extraction"
+    anchor: "attribution-windows-data-extraction"
+    content: |
+      {% include note.html first-line="**Pepperjam tables with attribution windows**" content="The info in this section only applies to the following tables:
+        - `creative_performance`
+        - `creative_performance_by_publisher`
+        - `publisher_performance`
+        - `transaction_details`
+        - `transaction_history` (28-day attribution window)" %}
+
+      When Stitch runs a replication job for {{ integration.display_name }}, it will use the value of the **Attribution Window** setting to query for and extract data for Incremental tables. An attribution window is a period of time for attributing data and the lookback period after those actions occur during which the data is counted.
```

Comment:
This is only applicable if the integration has an Attribution Window setting. Unlike Facebook Ads, Pepperjam *doesn't*. 

We talked about some other integrations that also have attribution windows, but don't have a setting in Stitch for it - d'you remember what they were? You could look at the docs for those integrations to get an idea of how to phrase this section.